pkg/timvir/core/components/NavigationFooter/NavigationFooter.tsx

```diff
@@ -80,9 +81,11 @@ function NavigationFooter(props: Props, ref: React.ForwardedRef<React.ElementRef
           className={css`
             display: flex;
             flex-direction: column;
-            align-items: flex-end;
             justify-content: flex-start;
-            text-align: right;
+            @media (min-width: 48rem) {
+              align-items: flex-end;
+              text-align: right;
+            }
```

Comment:
Why the change to right-align only at width>48rem?

Reply:
from this
![CleanShot 2024-05-29 at 10 16 24@2x](https://github.com/timvir/timvir/assets/6782188/3c246ce9-c11a-4501-9bf4-448b42e4d51e)

to this
![CleanShot 2024-05-29 at 10 17 00@2x](https://github.com/timvir/timvir/assets/6782188/78a07249-0e91-4f29-83ab-4565852425b0)